config/initializers/paperclip.rb

```diff
@@ -1 +1,15 @@
-Paperclip::DataUriAdapter.register
\ No newline at end of file
+Paperclip::DataUriAdapter.register
+
+module PatchPaperclipContentTypeDetector
```

Comment:
Este es un hack que encontré para patchear un problema con la identificación del type .xls (lo toma como x-ole-storage si no). Referencia https://github.com/mimemagicrb/mimemagic/issues/50

Reply:
Puede ser que x-ole-storage sea de openoffice.org?

Reply:
La verdad no estoy seguro a qué corresponde un `x-ole-storage`, de lo que busqué surge que es para `.msi` (https://www.whatisfileextension.com/msi/).

Reply:
Viste algo de ésto? https://github.com/mimemagicrb/mimemagic/issues/50#issuecomment-770251049

Reply:
https://github.com/lemenkov/shared-mime-info/blob/master/freedesktop.org.xml.in#L509-L525

Reply:
Investigué sobre esto, incluso llegué a bajarme el xml que mencionan y reemplazar el que tengo pero me sigue dando el mismo resultado.